ui/raidboss/data/06-ew/dungeon/stigma_dreamscape.ts

```diff
@@ -0,0 +1,257 @@
+      response: Responses.goLeft(),
+    },
+    {
+      id: 'Dreamscape Side Cannons Right',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6321', source: 'Proto-Omega', capture: false }),
+      response: Responses.goRight(),
+    },
+    {
+      id: 'Dreamscape Forward Interceptors',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6322', source: 'Proto-Omega', capture: false }),
+      response: Responses.getBehind(),
+    },
+    {
+      id: 'Dreamscape Rear Interceptors',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '6324', source: 'Proto-Omega', capture: false }),
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Go front of boss',
+        },
+      },
+    },
+    {
+      id: 'Dreamscape Chemical Missile',
+      type: 'HeadMarker',
+      netRegex: NetRegexes.headMarker({ id: '008B' }),
+      condition: Conditions.targetIsYou(),
+      response: Responses.spread(),
+    },
+    {
+      id: 'Dreamscape Electric Slide',
+      type: 'HeadMarker',
+      netRegex: NetRegexes.headMarker({ id: '0121' }),
+      infoText: (data, matches, output) => {
+        if (data.me === matches.target)
+          return output.target!();
+        return output.allies!({ target: matches.target });
+      },
+      outputStrings: {
+        target: {
+          en: 'Stack + Knockback on YOU!',
+        },
+        allies: {
+          en: 'Stack + knockback on ${target}',
+        },
+      },
+    },
+    {
+      id: 'Dreamscape Guided Missile',
+      type: 'Tether',
+      netRegex: NetRegexes.tether({ id: '0011' }),
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Guided Missile on YOU',
+        },
+      },
+    },
+    {
+      id: 'Dreamscape Mustard Bomb',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '632B', source: 'Proto-Omega' }),
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Dreamscape Assault Cannon',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '63AB', source: 'Arch-Lambda', capture: false }),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Go to second dash side',
+        },
+      },
+    },
+    {
+      id: 'Dreamscape Sniper Cannon',
+      type: 'HeadMarker',
+      netRegex: NetRegexes.headMarker({ id: ['004F', '0050', '0051', '0052'] }),
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, matches, output) => {
+        const limitCutNumber = limitCutNumberMap[matches.id];
+        return output.text!({ num: limitCutNumber });
+      },
+      outputStrings: {
+        text: {
+          en: '#${num} laser on YOU!',
```

Comment:
This might need rewording to indicate that the laser originates from the paired head marker and not from the boss?